extraction/text_extractor.py

```diff
@@ -0,0 +1,41 @@
+"""This script checks whether DOM has text tag or not and
+creates and returns the Text object"""
+
+import bs4
+from extraction.data_models.text import Text
+
+TEXT_TAGS = {'p', 'span', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'code'}
+HEADING_TAGS = {'h1', 'h2', 'h3', 'h4', 'h5', 'h6'}
+
+
+def empty_text(_str):
+    """Returns true is null string"""
+    return not _str and not _str.strip()
```

Comment:
Wouldn't checking only for _str.strip() be enough here?

Reply:
Actually if it's just only whitespaces in _str then _str.strip() leaves one whitespace. Atleast it happened if _str was one with blank spaces, one space was left. I will cross check and make the necessary changes.